src/metabase/api/collection.clj

```diff
@@ -267,13 +267,14 @@
    [:pinned-state {:optional true} [:maybe (into [:enum] (map keyword) valid-pinned-state-values)]]
    ;; when specified, only return results of this type.
    [:models       {:optional true} [:maybe [:set (into [:enum] (map keyword) valid-model-param-values)]]]
-   [:sort-info    {:optional true} [:maybe [:tuple
-                                            (into [:enum {:error/message "sort-columns"}]
-                                                  (map normalize-sort-choice)
-                                                  valid-sort-columns)
-                                            (into [:enum {:error/message "sort-direction"}]
-                                                  (map normalize-sort-choice)
-                                                  valid-sort-directions)]]]])
+   [:sort-info    {:optional true} [:maybe [:map
+                                            [:sort-column (into [:enum {:error/message "sort-columns"}]
+                                                                (map normalize-sort-choice)
+                                                                valid-sort-columns)]
+                                            [:sort-direction (into [:enum {:error/message "sort-direction"}]
+                                                                   (map normalize-sort-choice)
+                                                                   valid-sort-directions)]
+                                            [:official-collections-first? {:optional true} :boolean]]]]])
```

Comment:
It's too bad that this didn't catch the `?` thing. I feel a dilema between scattering mu/defn everywhere, vs the performance cost of it. Do you too?